src/entity/Game.ts

```diff
@@ -0,0 +1,32 @@
+import { Status } from '../types/game'
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToMany,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm'
+import { TournamentTeam } from './TournamentTeam'
+import { GameScore } from './GameScore'
+
+@Entity()
+export class Game {
+  @PrimaryGeneratedColumn('uuid')
+  id: string
+
+  @Column()
+  state: Status
+
+  @OneToOne(() => TournamentTeam)
+  @JoinColumn()
+  winner: TournamentTeam
+
+  @OneToOne(() => TournamentTeam)
+  @JoinColumn()
+  refTeam: TournamentTeam
+
+  @OneToMany(() => GameScore, (gameScore) => gameScore.game)
+  gameScores: GameScore
```

Comment:
This should be:

```typescript
  gameScores: GameScore[]
```

Also, you should also check that the first param of `@OneToMany` shouldn't be `() => GameScore[]`. I think what you have now is fine but can't remember off the top of my head.